flake.nix

```diff
@@ -111,10 +111,35 @@
         packages = {
           default = config.workspace.workspace;
           inherit (config.workspace) workspace;
+          local-telegraf = pkgs.writeShellApplication {
+            name = "local-telegraf";
+            runtimeInputs = [ pkgs.telegraf ];
+            text = ''
+              RUN_ID=$(jq --slurp --raw-output 'sort_by(.started_at|tonumber) | last | .run_id' < run_summary.jsonl)
+
+              sed --in-place "s/run_id = \"\"/run_id = \"$RUN_ID\"/" ./telegraf/local-telegraf.conf
+
+              echo "Running telegraf for run ID: $RUN_ID"
+              telegraf --config telegraf/local-telegraf.conf --once > >(tee logs/telegraf-stdout.log) 2> >(tee logs/telegraf-stderr.log >&2)
+
+              rm ./telegraf/metrics/*.influx
```

Comment:
Remove any metrics that have now been uploaded. Don't want to re-use/relabel any old host metrics.